src/Parser/TemplateParser.cs

```diff
@@ -217,22 +288,22 @@ private int ReadInt()
                 _position++;
                 i = i * 10 + digit;
             }
-            if (!hasDigits) 
+            if (!hasDigits)
                 throw new TemplateParserException("An integer is expected", _position);
-            
+
             SkipSpaces();
             return negative ? -i : i;
         }
 
-        private string ReadUntil(char search, bool required = true)
-        {
-            int start = _position;
-            int i = _template.IndexOf(search, _position);
-            if (i == -1 && required)
-                throw new TemplateParserException($"Reached end of template while expecting '{search}'.", _position);
-            _position = i == -1 ? _length : i;
-            return _template.Substring(start, _position - start);
-        }
+        //private string ReadUntil(char search, bool required = true)
```

Comment:
BTW, why keep the code behind comments?
The parser is now complete and we can always grab this from git history if needed.